pbr/pbr.py

```diff
@@ -359,6 +359,9 @@ def main():
                     os.path.join(out_cfg.depth_dir, filename) + ".exr",
                 )
 
+        # Check that the rotation matrix of the main camera is valid
+        print(cam_l.obj.matrix_world)
```

Comment:
If you want to keep this in, then add some text to tell the user what this matrix is.